scripts/dev/flashcrow.conf

```diff
@@ -1,6 +1,6 @@
 location /api {
```

Comment:
why do we even have this file?  we've never used `nginx` in local development...